### ISSUE TITLE
feat(scope): Allow Different Removal Actions for Packages in UAD-ng

### ISSUE THREAD

**Issue by rivanfebrian123** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#345]
## Feature Request: 

This request proposes a new feature for UAD-ng that allows specifying different removal actions (disable or uninstall) for individual packages within the JSON configuration files.

**Context:**

- A recent pull request (#341) highlighted an issue with `com.hoffnung`. Disabling this package is recommended, while uninstalling can lead to unintended consequences.
- The discussion identified a need for more granular control over removal actions for specific packages.

**Proposed Feature:**

Introduce a new field named `removalAction` within the JSON configuration for each package. This field will dictate the default action taken when a user attempts to remove the package through the recommended list.

**Benefits:**

- Improved user experience: Users can safely remove packages with specific actions (disable or uninstall) based on their needs.
- Enhanced safety: Prevents accidental uninstalls that might cause system instability.
- Increased flexibility: Allows for more nuanced control over package management.

**Implementation:**

- Update the JSON schema to include the `removalAction` field. This field can be a string with values like "disable" or "uninstall".
- Modify the application logic to interpret the `removalAction` field when processing user requests for package removal.
- Potentially update the user interface to reflect the different removal options for specific packages (optional).

**Example JSON Configuration:**

```json
"com.hoffnung": {
  "description": "TPMS\nRemote Config Test. Instead Uninstall better Disable this app\nbecause it uninstalling cause screen flicker",
  "removal": "Recommended",
  "removalAction": "Disable",
  "list": "Oem",
  "dependencies": [],
  "neededBy": [],
  "labels": []
},
"tech.palm.id": {
  "description": "Another package requiring disable instead of uninstall",
  "removal": "Recommended",
  "removalAction": "Disable",
  "list": "Oem",
  "dependencies": [],
  "neededBy": [],
  "labels": []
}
```

**Additional Considerations:**

-  Clear documentation on how to utilize the `removalAction` field.
-  User interface elements to visually differentiate between disable and uninstall options for specific packages (if applicable).

We believe this feature will significantly improve the user experience and safety when managing packages within UAD-ng.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Wakelock** [comment]
I completely agree. I need this feature for my pull: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/1433

**Comment by Rudxain** [comment]
Now that I think about it, wouldn't #1426 be better? Something like:
- "disable" becomes new default
- "Recommended" packs can be disabled or uninstalled anytime (we'll need to add both buttons, because a checkbox in settings is awkward)
- "Advanced" and higher (up to "Unsafe") cannot be Uninstalled unless Expert mode is enabled
- For Android 17 and higher, "Uninstall" is always hidden (or add a mechanism to detect which versions support it, for future-proofing)
